webi-docker.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+#shellcheck disable=SC2046
+
+VERSION_OF_UBUNTU_CONTAINER="22.04"
+WEBI_CONTAINER_FOLDER="${1:-""}"
+
+timestamp() {
+    echo "[+] $(date +'%F %T') [INFO] $*"
+    echo
+}
+
+timestamp_ending() {
+    echo "[+] $(date +'%F %T') [INFO] $*"
+}
+
+err() {
+    echo "[-] $(date +'%F %T') [ERROR] $*" >&2
+    echo
+}
+
+command_start() {
+    timestamp "Command $* has been started."
+    if ! "$@"; then
+        err "Command $* went wrong."
+        exit
+    fi
+    timestamp "Command $* has been ended."
+}
+
+remove_old_docker_containers() {
+    container="$(docker ps --all | grep -i Webi | awk '{print $1}')"
```

Comment:
This isn't part of shellcheck, but just my own style:

- can you do this without `awk` (perhaps with `cut`)?
  - awk is a complete, standalone scripting language, like JavaScript, and I don't want to include multiple languages (perl, python, awk, js, etc) in these scripts
- split up the subshell into multiple lines
  - where possible I try to do just one thing per line so that it's easier to read hte subscript

```sh
container="$(
    docker ps --all |
        grep -i webi |
        cut -d' ' -f 1
)"
```